system-parachains/asset-hubs/asset-hub-kusama/src/xcm_config.rs

```diff
@@ -314,8 +315,32 @@ pub type TrustedTeleporters = (
 	IsForeignConcreteAsset<FromSiblingParachain<parachain_info::Pallet<Runtime>>>,
 );
 
-/// We allow locations to alias into their own child locations.
-pub type Aliasers = AliasChildLocation;
+/// Defines all global consensus locations that the asset hub is allowed to alias into.
+pub struct MatchWhitelistedGlobalConsensus;
+impl Contains<Location> for MatchWhitelistedGlobalConsensus {
```

Comment:
I would be explicit with the naming:
```suggestion
pub struct PolkadotOrEthereumGlobalConsensus;
impl Contains<Location> for PolkadotOrEthereumGlobalConsensus {
```